dags/blocksec_plugin/ricochet_launchpad_price_operator.py

```diff
@@ -1,11 +1,8 @@
-from airflow.models.baseoperator import BaseOperator
 from airflow.utils.decorators import apply_defaults
-from blocksec_plugin.web3_hook import Web3Hook
-from blocksec_plugin.ethereum_wallet_hook import EthereumWalletHook
+from blocksec_plugin.abis import SHARE_PRICE_ABI
```

Comment:
Create a `REX_LAUNCHPAD_ABI` and use the actual launchpad contract ABI, find it here: 0x98d463A3F29F259E67176482eB15107F364c7E18